corehq/apps/sms/models.py

```diff
@@ -498,16 +498,30 @@ def outbound_backend(self):
 
 
 class SMS(SMSBase):
+    def to_json(self):
```

Comment:
fyi, we have django rest framework installed now which makes serialization very easy. here's an example: https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/serializers.py